src/openscm_runner/run.py

```diff
@@ -33,9 +50,9 @@ def run(
     output_variables : list[str]
         Variables to include in the output
 
-    full_config : bool
-        Include the configuration used to run each model in the output's
-        metadata
+    out_config : dict[str: tuple of str]
+        Dictionary where each key is a model and each value is a list of
```

Comment:
The typing states a tuple of str instead of a list

Reply:
```suggestion
        Dictionary where each key is a model and each value is a tuple of
```